news/2020-10-24-new-featured-artist-joe-ford.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "New Featured Artist: Joe Ford"
+date: 2020-10-24 09:00:00 +0000
+---
+
+We're proud to welcome **Joe Ford** aboard as our latest addition to the Featured Artist lineup!
+
+![](https://assets.ppy.sh/artists/109/header.jpg)
+
+Something of a rising star in the drum & bass scene, **Joe Ford** has been turning heads ever since 2011, drawing the support of big-name fixtures in the scene and making appearances on labels like *Hospital* and *Shogun*.
+
+Astute followers might've noticed that we've seen some of **Joe Ford**'s work in the Featured Artist program already, particularly his collaborative track with *Culprate* titled *Gaucho* under the *Unity Project* albums. 
+
+We're excited to announce that we're really filling that out today, with **38**new tracks now available featuring many of his signature hi-tech drum & bass hits.
+
+Check out a sample below:
+
+### Villain
+
+Check out [this 2016 challenge set](https://osu.ppy.sh/beatmapsets/399126) by [Yoges](https://osu.ppy.sh/users/3821342)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/109/previews/2885.mp3" type="audio/mpeg">
+</audio>
+
+### Culprate & Joe Ford - Gaucho
+
+Check out [this collaborative set](https://osu.ppy.sh/beatmapsets/847776) from the original Mappers' Guild members hosted by [ScubDomino](https://osu.ppy.sh/users/8972308), [this osu!taiko set](https://osu.ppy.sh/beatmapsets/936234) by [Nifty](https://osu.ppy.sh/users/4956097) and [this osu!taiko set](https://osu.ppy.sh/beatmapsets/937645) by [vnrl](https://osu.ppy.sh/users/4799788)!
```

Comment:
it's ``vrnl`` not ``vnrl`` :(

![image](https://user-images.githubusercontent.com/36564236/97077836-266f1000-1611-11eb-9641-2f2cc814b971.png)